meinberlin/apps/projects/views.py

```diff
@@ -127,6 +127,9 @@ def get_queryset(self):
                 Q(participants__pk=self.request.user.pk) |
                 Q(organisation__initiators__pk=self.request.user.pk) |
                 Q(moderators__pk=self.request.user.pk)
+            ) & (
+                Q(is_archived=False) |
+                Q(externalproject__bplan=None)
```

Comment:
This code deserves a comment.

Reply:
i just looked over this again after my holidays and am not sure if it does really work is_archived is also set to True via a filter.
i would like to look into it again. so please don't merge yet

Reply:
okay the expression is alright. all hail to the glory of databases :D